homeassistant/components/elkm1/config_flow.py

```diff
@@ -116,7 +103,7 @@ async def async_step_user(self, user_input=None):
             try:
                 info = await validate_input(user_input)
 
-            except CannotConnect:
+            except exceptions.ConfigEntryNotReady:
```

Comment:
this doeant seem right to me to be using this error in config flow

Reply:
I agree, we should let the `asyncio.TimeoutError` raise and catch it here and in the place we want to throw `ConfigEntryNotReady`